fabcar/startFabric.sh

```diff
@@ -109,8 +109,25 @@ Go:
   Start by changing into the "go" directory:
     cd go
 
+  Next, set hosts:
+    sudo vim /etc/hosts
+
+  add Under the content into  /etc/hosts
+    ip	peer0.org1.example.com  
+    ip	peer0.org2.example.com  
+    ip	peer0.org2.example.com  
+
+    for example:
+    127.0.0.1       peer0.org1.example.com  peer0.org2.example.com  orderer.example.com
+
+  Next, set environment variable:
+    export DISCOVERY_AS_LOCALHOST=true
```

Comment:
A description like this will inform the user, since you have a export command in line 127

```suggestion
  Set environment variable DISCOVERY_AS_LOCALHOST to true, to indicate to ServiceDiscovery that all hostnames maps to localhost
```